src/analysis/special_functions/gaussian.lean

```diff
@@ -17,6 +18,30 @@ noncomputable theory
 open real set measure_theory filter asymptotics
 open_locale real topological_space
 
+open complex (hiding exp continuous_exp abs_of_nonneg)
+notation `cexp` := complex.exp
+
+/-- If `f, g` are continuous functions `0 < re z`, and `(f z) ^ 2 = (g z) ^ 2` holds on this region,
+then as soon as `f 1 = g 1` we have `f z = g z` for all `z`. -/
+lemma eq_of_sq_eq_of_continuous {f g : ℂ → ℂ}
```

Comment:
I've pushed the most general version I could think of (for functions from an arbitrary topological space to a field with a T1 topology); but it's not clear to me where in the library this should go. It won't fit in `topology/connected.lean` because `t1_space` is not available there. Do you have any suggestions?

Reply:
It looks like it could go to `topology/algebra/field.lean`, for instance.